tests/testthat/error/test-subscript-loc.txt

```diff
@@ -66,17 +70,17 @@ x `i` has the wrong type `data.frame<
   gear: double
   carb: double
 >`.
-i These indices must be indicators, locations or names.
+i The subscript must contain indicators, locations or names.
```

Comment:
I think "be" sounds better to me. Also needs Oxford comma.

Reply:
Or maybe tie back to header? "A valid subscript vector must be logical, numeric, or character."